recipes/ptflops/conda_build_config.yaml

```diff
@@ -0,0 +1,3 @@
+channels:
```

Comment:
conda-forge packages should not depend on other non-conda-forge channels. Please remove this. I believe all of the dependencies are available on the `conda-forge` channel.